projects/plugins/crm/webpack.config.js

```diff
@@ -120,6 +120,9 @@ const crmWebpackConfig = {
 	},
 	resolve: {
 		...jetpackWebpackConfig.resolve,
+		alias: {
+			"crm": path.resolve(__dirname, "src/js/")	
+		}
```

Comment:
We're potentially overwriting Jetpack Webpack Config's `alias` property this way, can we instead extend it using something like this: https://github.com/Automattic/jetpack/blob/0bc1ca706c403d13b91ed5fbd9831cb6a6c219d0/projects/packages/search/tools/webpack.instant.config.js#L47 ?